tests/functional/test_pkgutil.py

```diff
@@ -53,7 +54,8 @@ def _read_results_file(filename):
 def test_pkgutil_iter_modules(package, script_dir, tmpdir, pyi_builder,
                               archive):
     # Ensure package is available
-    pytest.importorskip(package)
+    if not importable(package):
```

Comment:
Given the structure of the test, this could probably check for availability of the top-level package to avoid triggering the submodule/subpackage check (which should be unnecessary).

Reply:
Like this?

Reply:
Yeah, that should do the trick.